pages/data-providers/[data-provider-id]/overview.css

```diff
@@ -19,3 +19,13 @@
     grid-template-columns: repeat(3, 1fr);
   }
 }
+
+.overview-card {
+  display: flex;
+  flex-direction: column;
+}
+
+.browse-button {
+  align-self: flex-start;
```

Comment:
You might wanted to say `flex-end`. However, it's not needed anyway when you use `margin-top: auto`. You need only one of these. I prefer `align-self: flex-end`.

Reply:
`flex-end` would move the button to the right. It defines alignment along with cross not main axis.